certified-connectors/SkyPointCloud/apiDefinition.swagger.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "SkyPoint Cloud",
-    "description": "SkyPoint is a customer data platform that champions privacy and compliance through connected privacy-first customer experiences. SkyPoint Connector provides a set of APIs to work with SkyPoint platform instances and entities.",
+    "description": "SkyPoint Cloud is the industry's first Modern Data Stack Platform with built-in data lakehouse, account 360, customer 360, identity resolution, data privacy vault, ELT / Reverse ETL, data integration, privacy compliance automation, data governance, analytics, partner integrations, and managed services for organizations in several industries including healthcare, life sciences, senior living, retail, hospitality, business services, and financial services. SkyPoint Connector provides a set of APIs to work with SkyPoint Modern Data Stack Platform solutions",
```

Comment:
@sjaishanker The 'description' value must be between '30' and '500' characters. thank you so much

Reply:
Sure @Amjed-Ayoub, will make the changes. Thanks